modules/xfem/src/userobjects/CircleCutUserObject.C

```diff
@@ -67,3 +67,22 @@ CircleCutUserObject::isInsideCutPlane(Point p) const
     return true;
   return false;
 }
+
+const std::vector<Point>
+CircleCutUserObject::getCrackFrontPoints(unsigned int number_crack_front_points) const
+{
+  std::vector<Point> crack_front_points(number_crack_front_points);
+  Point v1 = _vertices[0] - _center;
+  Xfem::normalizePoint(v1);
+  Point v2 = _normal.cross(v1);
+  Xfem::normalizePoint(v2);
```

Comment:
v2.norm() - This already exists as a member.

Reply:
I will use it. Thanks.